46.md

```diff
@@ -2,161 +2,83 @@ NIP-46
 ======
 
 Nostr Connect
-------------------------
+-------------
 
 `draft` `optional`
 
-## Rationale
+This NIP describes a method for 2-way communication between a **remote signer** and a normal Nostr client. The remote signer could be, for example, a hardware device dedicated to signing Nostr events, while the client is a normal Nostr client.
 
-Private keys should be exposed to as few systems - apps, operating systems, devices - as possible as each system adds to the attack surface.
+## Signer Discovery
 
-Entering private keys can also be annoying and requires exposing them to even more systems such as the operating system's clipboard that might be monitored by malicious apps.
+The client must somehow be able to contact the signer through a common relay.
```

Comment:
`The client always starts by generating a random key which is used to communicate with the signer`

Reply:
no, based on this flow the signer will create the client's random key. also it should be possible (but not required) to reuse this random key for future sessions.